runtime/compiler/control/CompilationThread.cpp

```diff
@@ -4622,17 +4765,20 @@ TR::CompilationInfo::addMethodToBeCompiled(TR::IlGeneratorMethodDetails & detail
          {
          // Must find one that is SUSPENDED/SUSPENDING
          TR::CompilationInfoPerThread *compInfoPT = getFirstSuspendedCompilationThread();
-         compInfoPT->resumeCompilationThread();
-         if (TR::Options::getCmdLineOptions()->getVerboseOption(TR_VerboseCompilationThreads))
+         if (compInfoPT) // FIXME this is null sometimes during shutdown. Why??
```

Comment:
Is this FIXME specific to JITServer? The existing code seems to assume it will always be non-NULL. If it's just JITServer maybe we should wrap the `if (compInfoPT)` in an `#ifdef` so that we don't hide a potential bug in non-JITServer mode, just in case we don't come back to this FIXME any time soon.